lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -29,6 +29,9 @@ static int64_t getDtypeIntegerFromMlirType(Type dtype) {
   if (dtype.isa<Float32Type>())
     return 6;
 
+  if (dtype.isa<BFloat16Type>())
```

Comment:
nit: can we use the torch_upstream constants instead of raw integers throughout this function? (separate patch OK)

Reply:
PR https://github.com/llvm/torch-mlir/pull/1020 updates the code to make use of the `ScalarType` enum.